### ISSUE TITLE
Is a dataset for validation not provided?

### ISSUE THREAD

**Issue by jeonghyunkeem** [skanti/Scan2CAD#20]
Hello, I'm working on training the network to predict cad alignment on RGB-D scan. In my understaning, the annotation for testset is not provided to prevent the parameter tuning on testset. But is it the same with validation set, either? In main.py script, it seems that a scene list for validation is required but I cannot find it.  How can I handle overfitting on train set?
